pkgs/applications/science/math/mxnet/default.nix

```diff
@@ -60,6 +62,12 @@ stdenv.mkDerivation rec {
     rm "$out"/lib/*.a
   '';
 
+  # used to mark cudaSupport in python310Packages.mxnet as broken;
+  # other attributes exposed for consistency
+  passthru = {
+    inherit cudaSupport cudnnSupport cudatoolkit cudnn;
```

Comment:
CC @abbradar 

cmake fails to configure in python 3.10 when enabling cuda and python